static/scss/user-chip.scss

```diff
@@ -70,3 +71,8 @@
     display: block;
   }
 }
+
+.learner-result:hover {
+  background: $bg-light-gray;
+}
```

Comment:
could you move this into `search-page.scss`, where the rest of the styling for `.learner-result` is?